.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+  - '3.6'
+cache:
+  pip: true
+  directories:
+    - /home/travis/.yarn-cache/
+install: pip install pytest "jupyterlab<1.0.0"
+script:
+  - yarn
+  - yarn build
+  - pip install .
+  - jupyter serverextension enable --py jupyterlab_pullrequests
+  - jupyter labextension install .
+  - pytest tests
+  - yarn run lint
```

Comment:
Consider logically grouping these commands

- All yarn/frontend validations
- All python/backend tests/validations
- Sanity test for installing the extension